assets/css/_diversion_page.scss

```
@@ -21,6 +25,33 @@
     }
   }
 
+  &--mobile,
+  &--mobile_landscape_tablet_portrait {
+    grid-template:
+      "heading" min-content
+      "panel" 0
+      "map" auto
```

Hmm, if we're not using panel anymore, let's change the name so it fits semantically?
```suggestion
      "map-and-panel" auto
```